internal/build/generate.go

```diff
@@ -200,20 +270,26 @@ var y = [{{len .Zones}}]Zone{
 	{{range $d := .Domains}} \
 		{{$z := (index $.Zones $d)}} \
 		{ \
-			{{quotedDomain $d}}, \
+			{{domainString $d}}, \
 			{{if $z.IsIDN}}/* {{$d}} */{{end }} \
 			{{if $z.ParentDomain}} &z[{{$z.ParentOffset}}] {{else}} r {{end}}, \
 			{{if $z.SubdomainsEnd}} z[{{$z.SubdomainsOffset}}:{{$z.SubdomainsEnd}}] {{else}} x {{end}}, \
-			{{if $z.NameServers}} s{ {{range $z.NameServers}}{{quotedDomain .}},{{end}}} {{else}} n {{end}}, \
-			{{if $z.Wildcards}} s{ {{range $z.Wildcards}}{{quoted .}},{{end}}} {{else}} n {{end}}, \
-			{{if $z.Locations}} s{ {{range $z.Locations}}{{quoted .}},{{end}}} {{else}} n {{end}}, \
-			{{if $z.Languages}} s{ {{range $z.Languages}}{{quoted .}},{{end}}} {{else}} n {{end}}, \
-			{{quotedDomain $z.WhoisServer}}, \
-			{{quotedURL $z.WhoisURL}}, \
-			{{quotedURL $z.InfoURL}}, \
+			{{domainSlice $z.NameServers}}, \
+			{{domainSlice $z.Wildcards}}, \
```

Comment:
In the removed code `Wildcards` was not treated as a domain, it used `quoted` not `quotedDomain`, but in this change `Wildcards` uses `domainSlice` and not `stringSlice`. Is that inentional?

Reply:
Wildcards are either IP addresses or hostnames, so should be escaped/normalized as domain names.